### PR TITLE
Don't overwrite dashboard template

### PR DESCRIPTION
**Pull request by afoerster**
- Hardcode template name in script

### CODE REVIEW

File: visualization/arcadia/README.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ Arcadia is visual analytics and BI software that runs natively within modern dat
 
 
 ### Replace dataset_name and dataset_detail from `pulse.json`
-- Use `replace_dataset_name.sh` script to replace dataset_name and dataset_detail in pulse.json
+- Use `replace-dataset-name.sh.sh` script to replace dataset_name and dataset_detail in pulse.json
```
Comment (andahme) [inline visualization/arcadia/README.md]: I think this is likely an oversight (.sh.sh)

File: visualization/arcadia/README.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ Arcadia is visual analytics and BI software that runs natively within modern dat
...
 - For the new dataset created in the last step, dataset_name is `new_solr_test_dataset` and dataset_detail is `Solr.logging-pulse-test_all`
-- command `./replace_dataset_name.sh pulse.json new_dataset_name new_dataset_detail`
-- `pulse.json` is ready to be imported
+- command `./replace-dataset-name.sh.sh new_dataset_name dataset_source`
```
Comment (andahme) [inline visualization/arcadia/README.md]: Same (.sh.sh)

File: visualization/arcadia/README.md (hunk 1 above)
Comment (afoerster) [reply visualization/arcadia/README.md]: Ideed, fixed.

File: visualization/arcadia/README.md (hunk 2 above)
Comment (afoerster) [reply visualization/arcadia/README.md]: Done.

**Comment by afoerster** [comment]
Closing and adding to the new documentation (mkdocs)

**Pull request closed without merging**